lib/daru/index/index.rb

```diff
@@ -222,6 +222,44 @@ def reorder(new_order)
       self.class.new(new_order.map { |i| from[i] })
     end
 
+    # Sorts a `Index` according to its values. Defaults to ascending order sorting.
+    # raises ArgumentError when all the values are not of the same class.
+    #
+    # == Options
+    #
+    # * +:ascending+ - if false, will sort in descending order. Defaults to true.
+    #
+    # * +:get_new_order+ - if true, then method will return the new order (in list) of
```

Comment:
You should drop it. No other Ruby collection provides this fucntionality, and it is hard to justify. YAGNI is a pretty useful principle.